vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -1,141 +1,75 @@
-
-## Create the Project Structure
-First, create a parent project directory (e.g. `example-bundle`) along with a child bundle directory. In a project generated by the [Entando Component Generator](../../../docs/create/component-gen-overview.md), the microservice and micro frontend source files live under the parent directory.
-
-``` sh
-mkdir -p example-bundle/bundle; cd example-bundle/bundle
-```
-## Add a Simple Widget
-
-Create a widget directory
-``` sh
-mkdir widgets
-```
-
-Create a widget descriptor file within that directory
-``` sh
-touch widgets/example-widget.yaml
-```
-
-Populate the widget descriptor file `example-widget.yaml` with a simple definition. Make sure to retain the correct YAML indentation of 2 or 4 spaces.
-``` yaml
-code: example-widget
-titles:
-   en: Example Widget
-   it: Widget d'esempio
-group: free
-customUi: <h2>Hi from Example Widget</h2>
-```
-
-## Create the Bundle Descriptor
-
-The main file processed by the Entando Component Repository is `descriptor.yaml`, which describes all of the components within the bundle. The name of the bundle descriptor file must be `descriptor.yaml` and it must be stored in the child bundle directory (e.g. `example-bundle/bundle`).
-```sh
-touch descriptor.yaml
-```
-
-Populate the bundle descriptor file with the following YAML definition
-``` yaml
-code: example-bundle
-description: This is an example of an Entando bundle
-components:
-  widgets:
-    - widgets/example-widget.yaml
-```
-Component descriptor file names and locations (e.g. `widgets/example-widget.yaml`) are arbitrary since the bundle descriptor explicitly points to those files. Convention is to group components by type with all widgets in one directory, all page templates in another, etc.
-
-## Publish the Bundle
-
-The bundle can be published using the CLI or the steps can be performed manually.
-
-### CLI Steps
-1. Change to the project directory if needed
-```sh
-cd example-bundle
-```
-
-2. Initialize the Entando project and accept the defaults
-``` sh
-ent prj init
-```
-
-3. Initialize the publication system. This step requires the empty Git repository URL (ending in .git) and your Git credentials.
-``` sh
-ent prj pbs-init
-```
+This tutorial describes how to build and publish a single-component Entando Bundle consisting of a simple widget. The step sequence below performs the following functions:
 
-4. Publish the bundle to Git. By convention, the first version is assigned the tag `v0.0.1`, but the prefix "v" is optional.
-``` sh
-ent prj pbs-publish
-```
-Running just the command `ent prj pbs-publish` will quickly push subsequent iterations of the bundle to Git. You will be asked to input the bundle version each time. You must be consistent with versioning format and alphanumeric precedence to ensure that iterations are listed in the correct order. 
+1. Define the bundle component
+2. Check the bundle artifacts into a Docker repository
```

Comment:
"check in" is a git term, probably "push the bundle artifact to a Docker registry" is more typical.